source/AI.cpp

```diff
@@ -197,7 +197,7 @@ namespace {
 	bool ShouldRefuel(const Ship &ship, const DistanceMap &route, double fuelCapacity = 0.)
 	{
 		if(!fuelCapacity)
-			fuelCapacity = ship.Attributes().Get("fuel capacity");
+			fuelCapacity = ship.Attributes().Get("fuel capacity"_fnv1a);
```

Comment:
Is it really necessary to have to append `_fnv1a` to every attribute name that we want to access now? Can the hash part not be something that `Get` handles?

Reply:
I fear yes: appending "_fnv1a" means you are calling a function (`operator""`) and that member needs also to be called at compile time.

Put another way: the `Get` method is receiving an `HashWrapper` as a parameter, not a `char *`.

But if anyone know about a workaround I will do.

Reply:
BTW: As per comment review, I'm changing "_fnv1a" -> "_hash"

Reply:
@Amazinite; @Hurleveur; @quyykk : I'm realizing I'm introducing something a bit new in this codebase, so are you all fine with this or do you prefer me to stick on "standard", eg:

`ship.Attributes().Get(HashOf("fuel capacity"));`

Reply:
I prefer using the "standard" HashOf(); it seems more readable/more clear.